docs/content/catalog.md

```diff
@@ -1,7 +1,10 @@
 ---
```

Comment:
I see that the file got moved out of the `common` directory. It should actually stay in that directory. Can you please move the file back?

Reply:
I moved it to doc on purpose since it seems that currently there is only once concept(e.g. catalog), and I feel better to put it in docs folder so that its position it is quite close to table. What do you think?

Reply:
it should stay under `common` IMO. We just need to make sure we properly link to that page

Reply:
Cool, I'll fix that